OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -200,18 +208,21 @@ bool CaptureClient::TryAbortCapture() {
   return true;
 }
 
-void CaptureClient::FinishCapture() {
+ErrorMessageOr<void> CaptureClient::FinishCapture() {
   if (reader_writer_ == nullptr) {
-    return;
+    return outcome::success();
   }
 
   grpc::Status status = reader_writer_->Finish();
+  ErrorMessageOr<void> result = outcome::success();
   if (!status.ok()) {
```

Comment:
maybe just move this check to line 227, in which case you don't need temporary result variable here.

Reply:
Yep, done.